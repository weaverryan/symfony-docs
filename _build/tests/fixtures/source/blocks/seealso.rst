
.. seealso:

    Also check out the homepage
