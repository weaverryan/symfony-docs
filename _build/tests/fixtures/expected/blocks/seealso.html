<!DOCTYPE html>
<html>
<head>
    <meta charset="utf-8" />
</head>
<body>
<div class="admonition-wrapper">
    <div class="seealso"></div>
    <div class="admonition admonition-seealso">Also check out the homepage</div>
</div>
</body>
</html>